tutorials/data-lake-sof-multiple-directories/data-lake-sof-multiple-directories.md

```diff
@@ -0,0 +1,238 @@
+Let us CREATE THE TABLE now
+
+```SQL
+CREATE TABLE HDLADMIN_TPCH_SQLONFILES.HDLRE_ORDERS_SQLONFILES
+(
+ORDERYEAR		varchar(4),
+ORDERMONTH	varchar(2),
+O_ORDERKEY           bigint,
+O_CUSTKEY              integer,
+O_ORDERSTATUS     varchar(2),
+O_TOTALPRICE         decimal(15,2),
+O_ORDERDATE          date,
+O_ORDERPRIORITY   varchar(15),
+O_CLERK                    varchar(15),
+O_SHIPPRIORITY       integer,
+O_COMMENT             varchar(79)   
+) IN FILES_SERVICE;
+```
+<!-- border --> ![DBX Screenshot](image-1.png)
+
+For information about the parameter definitions and supported data types, see  [CREATE (Remote) TABLE Statement for Data Lake Relational Engine (HANA DB-Managed)](help.sap.com/docs/SAP_HANA_DATA_LAKE/a898e08b84f21015969fa437e89860c8/24e694b566814ad285cb32fe3e5d3928.html?state=DRAFT&version=2022_1_QRC)
+
+
+Next, we create a virtual table. Notice in the SQL below where the remote servers name goes and where the reference to the table in the Files Service goes. Over here, we will be creating an ORDERS VIRTUAL TABLE  in HDLRE that points to the ORDERS table that we just created in SQL On Files service.
+
+The EXISTING keyword is used in conjunction with CREATE TABLE and IN FILES SERVICE to indicate that a table exists remotely in SQL on Files and to import its metadata into the new virtual table. The remote table is now visible to users thanks to this syntax.
+
+`Note that the table definition will need to match the expected schema/data types of the columns in the structured file.`
+
+```SQL
+CREATE EXISTING TABLE ORDERS_SQLONFILES
+ (
+ORDERYEAR			varchar(4) ,
+ORDERMONTH		varchar(2),
+O_ORDERKEY		bigint ,
+O_CUSTKEY			integer ,
+O_ORDERSTATUS		varchar(2) ,
+O_TOTALPRICE		decimal(15,2) ,
+O_ORDERDATE 		date ,
+O_ORDERPRIORITY  	varchar(15) ,
+O_CLERK 			varchar(15) ,
+O_SHIPPRIORITY		integer ,
+O_COMMENT			varchar(79) ,
+) AT 'sof..HDLADMIN_TPCH_SQLONFILES.HDLRE_ORDERS_SQLONFILES';
+```
+
+<!-- border --> ![DBX Screenshot](image-2.png)
+
+
+
+
+
+### STEP 2: Optimize SQL on Files Performance Using Folder Hierarchies
+
+Another way to declare a data source with SQL on Files is using folder hierarchies. Folder hierarchies allow you to skip (or prune) files that do not need to be parsed in a query. Take a look at the following example.
+
+<!-- border --> ![Folder hierarchies](image-3.png)
+
+
+This diagram represents the folder structure hierarchy of the ORDERS table, for years 2020 to 2022 for ORDERS table there will be yearly data and from 2023 onwards to 2025, the table will have monthly data as well.
+In this case, we will create the folder structure in such a way that for ORDERYEAR=2020, it will query the data for that particular year i.e. 2020. We can also query the data for a particular month from that year and that is when ORDERMONTH and ORDERYEAR parameters comes into role. `SELECT * FROM ORDERS WHERE ORDERYEAR=2023 AND ORDERMONTH=05;` to skip parsing any data files under all the other directories for better performance.
```

Comment:
Please don't use the "royal we" when addressing the user. Use "you" instead.
Please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtvf24ta2) for details.